serverscommands/instancecommands/list.go

```diff
@@ -92,11 +97,28 @@ func tableList(c *cli.Context, i interface{}) {
 	t := tablewriter.NewWriter(c.App.Writer)
 	t.SetAlignment(tablewriter.ALIGN_LEFT)
 	keys := []string{"ID", "Name", "Status", "Public IPv4", "Private IPv4", "Image", "Flavor"}
-	t.SetHeader(keys)
+	if c.IsSet("noborder") {
+		t.SetBorder(false)
+		t.SetCenterSeparator("")
+		t.SetRowSeparator("")
+		t.SetColumnSeparator("")
```

Comment:
Yesterday I added these across all the services (bluh copy pasta) then realized that there wasn't a way to make a clean plain table with tablewriter.

I ended up switching to tabwriter.